test/kata3.numberToReversedDigits.test.js

```diff
@@ -1,5 +1,6 @@
 const { numberToReversedDigits } = require("../src");
 
 describe("numberToReversedDigits", () => {
-  test("returns a reversed array of the number's digits", () => {});
+  it("returns a reversed array of the number's digits", () => {});
+  expect(numberToReversedDigits(4321)).toEqual([1,2,3,4])
```

Comment:
not testing anything as the `expect` statement is outside of the callback passed to `it`